docs/pages/blog/turbo-1-1-0.mdx

```diff
@@ -0,0 +1,143 @@
+- Including environment variables in `globalDependencies` list prefixed by a `$` will impact the cache fingerprint of _all_ tasks.
+- Including files or globs of files in `globalDependencies` will impact the cache fingerprint of _all_ tasks.
+- The value of any environment variable that includes `THASH` in its name will impact the cache fingerprint of _all_ tasks.
+
+```jsonc
+{
+    "pipeline": {
+      "build": {
+        "dependsOn": {
+          "^build"
+          // env vars will impact hashes of all "build" tasks
+          "$SOME_ENV_VAR"
+        },
+        "outputs": ["dist/**"]
+      },
+      "web#build": { // override settings for the "build" task for the "web" app
+        "dependsOn": [
+          "^build",
+           // env vars that will impact the hash of "build" task for only "web" app
+           "$STRIPE_SECRET_KEY",
+           "$NEXT_PUBLIC_STRIPE_PUBLIC_KEY",
+           "$NEXT_PUBLIC_ANALYTICS_ID",
+        ],
+        "outputs": [".next/**"],
+      },
+      "docs#build": { // override settings for the "build" task for the "docs" app
+        "dependsOn": [
+          "^build",
+           // env vars that will impact the hash of "build" task for only "web" app
+           "$STRIPE_SECRET_KEY",
+           "$NEXT_PUBLIC_STRIPE_PUBLIC_KEY",
+           "$NEXT_PUBLIC_ANALYTICS_ID",
+        ],
+        "outputs": [".next/**"],
+      }
+    },
+    "baseBranch": "origin/main",
+    "globalDependencies": [
+      "$GITHUB_TOKEN"// env var that will impact the hashes of all tasks,
+      "tsconfig.json" // file contents will impact the hashes of all tasks,
+      ".env.*" // glob file contents will impact the hashes of all tasks,
+    ],
+  }
+}
+```
+
+Note: In most monorepos, you don't often use environment variables in shared packages, but mostly in applications. Thus, to get higher cache hit rates, you should only include environment variables in the app-specific tasks where they are used/inlined.
+
+For more information, read the [caching and hashing documentation](https://turborepo.org/docs/features/caching)
+
+## Partial Yarn v2/v3 support
+
+In addition to Yarn v1, npm, and pnpm package managers, Turborepo now supports Yarn v2+ with [`nodeLinker: "node-modules"` set in `.yarnrc.yml`](https://yarnpkg.com/configuration/yarnrc#nodeLinker). This key tells Yarn v2+ to mimic Yarn v1's `node_modules` installation behavior. Yarn v2+ Plug'n'Play (a.k.a. “PnP”) is not currently supported.
+
+## What's next?
+
+[Since launch](https://turborepo.org/blog/joining-vercel), Turborepo has focused on seamless incremental adoption/migration and speeding up CI/CD. We are committed to both of those values, but now we'll also be focusing on improving Turborepo's day-to-day ergonomics for local development and observability. We're really excited about this next chapter and will be sharing more details soon.
```

Comment:
```suggestion
[Since our launch](https://turborepo.org/blog/joining-vercel), Turborepo has focused on seamless incremental adoption/migration and speeding up CI/CD. We are committed to both of those values, but now we'll also be focusing on improving Turborepo's day-to-day ergonomics for local development and observability. We're really excited about this next chapter and will be sharing more details soon.
```